cypress/e2e/WebInterface/Test Cases/QI-CORE Test Case/TestCaseExportImport/TestCaseImportValidations.cy.ts

```diff
@@ -681,10 +705,11 @@ describe('Test case uniqueness error validation', () => {
         //verifies alert message at top of page informing user that no test case was imported
         Utilities.waitForElementVisible(TestCasesPage.importTestCaseAlertMessage, 35000)
         cy.get(TestCasesPage.importTestCaseAlertMessage).find('[id="content"]').should('contain.text', '(0) test case(s) were imported. The following (1) test case(s) could not be imported. Please ensure that your formatting is correct and try again.')
-        cy.get('[id="content"]').find('[data-testid="failed-test-cases"]').find('[class="sc-hCPjZK iYkvOm"]').should('contain.text', 'Reason: The Test Case Group and Title are already used in another test case on this measure. The combination must be unique (case insensitive, spaces ignored) across all test cases associated with the measure.')
+        cy.get('[id="content"]').find('[data-testid="failed-test-cases"]').find('span').should('contain.text', 'Reason: The Test Case Group and Title are already used in another test case on this measure. The combination must be unique (case insensitive, spaces ignored) across all test cases associated with the measure.')
```

Comment:
The class value here is generated as part of the build process, we would need to update this with every regression cycle. Using the html tag of `span` here instead gets us the same element without the need to constantly update values.